src/Microsoft.DotNet.ProjectJsonMigration/ConstantPackageNames.cs

```diff
@@ -6,5 +6,6 @@ namespace Microsoft.DotNet.ProjectJsonMigration
     internal class ConstantPackageNames
     {
         public const string CSdkPackageName = "Microsoft.NET.Sdk";
+        public const string CWebSdkPackageName = "Microsoft.NET.Sdk.Web";
```

Comment:
These names don't feel right. I know this PR isn't introducing the pattern, but we'll want to take another look.